internal/registry/registry.go

```diff
@@ -16,38 +17,40 @@ import (
 // imports and ensures there are no conflicts in the imported package
 // qualifiers.
 type Registry struct {
-	srcPkg     *packages.Package
-	moqPkgPath string
-	aliases    map[string]string
-	imports    map[string]*Package
+	srcPkgName  string
+	srcPkgTypes *types.Package
+	moqPkgPath  string
+	aliases     map[string]string
+	imports     map[string]*Package
 }
 
 // New loads the source package info and returns a new instance of
 // Registry.
 func New(srcDir, moqPkg string) (*Registry, error) {
 	srcPkg, err := pkgInfoFromPath(
-		srcDir, packages.NeedName|packages.NeedSyntax|packages.NeedTypes|packages.NeedTypesInfo|packages.NeedDeps,
+		srcDir, packages.NeedName|packages.NeedSyntax|packages.NeedTypes,
```

Comment:
From the [docs](https://pkg.go.dev/golang.org/x/tools/go/packages#LoadMode):

> // NeedTypesInfo adds TypesInfo.
    NeedTypesInfo

> // NeedDeps adds the fields requested by the LoadMode in the packages in Imports.
   NeedDeps

Neither `TypesInfo` or `Imports`  filed is used, therefore there is no need to load this information.